contrib/x264/module.defs

```diff
@@ -1,7 +1,9 @@
 $(eval $(call import.MODULE.defs,X264,x264,YASM PTHREADW32))
 $(eval $(call import.CONTRIB.defs,X264))
 
+# TODO: Unknown upstream url
```

Comment:
As far as I know, there is no such thing.  They don't tag releases and don't appear to post release source archives anywhere.  I think our best bet would be to settle on syncing with a nightly snapshot

ftp://ftp.videolan.org/pub/videolan/x264/snapshots/